integration/kubernetes/k8s-liveness-probes.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	pod_name="liveness-exec"
+}
+
+@test "Liveness probe" {
+	wait_time=120
+	sleep_time=5
+	sleep_liveness=7
+
+	# Create pod
+	sudo -E kubectl create -f pod-liveness.yaml
+
+	# Check pod creation
+	pod_status_cmd="sudo -E kubectl get pods -a | grep $pod_name | grep Running"
+	waitForProcess "$wait_time" "$sleep_time" "$pod_status_cmd"
+
+	# Check liveness probe returns a success code
+	sudo -E kubectl describe pod $pod_name | grep -E "Liveness|#success=1"
+
+	# Sleep necessary to check liveness probe returns a failure code
+	sleep $sleep_liveness
+	sudo -E kubectl describe pod $pod_name | grep "Liveness probe failed"
+}
+
+teardown() {
+	sudo -E kubectl delete deployment "$pod_name"
+	# Wait for the pods to be deleted
```

Comment:
in k8s 1.12, we can safely remove this as the command waits until the pod is deleted.